src/sentry/models/project.py

```diff
@@ -349,6 +350,21 @@ def transfer_to(self, organization):
                 environment_id=Environment.get_or_create(self, environment_names[environment_id]).id
             )
 
+        # Manually move over organization id's for Monitors
+        try:
+            Monitor.objects.filter(organization_id=old_org_id).update(
+                organization_id=organization.id
+            )
+        except IntegrityError as e:
+            logging.exception(
+                "Failed to transfer monitors to new organization.",
+                extra={
+                    "error": str(e),
+                    "organization_to": organization.id,
+                    "organization_from": old_org_id,
+                },
+            )
```

Comment:
yeah this wouldn't even work because the atomic transaction gets mad. need to figure out the right approach here. don't want to iterate through the monitors individually but not sure what options I have. plus the monitor environments I guess.

Reply:
maybe i'll just delete. still doesn't seem quite right....need to think about how it interfaces with upsert

Reply:
I think we should just delete all monitors. it's really messy otherwise. upsert should fix everything with DSN's generally.

also the `transaction.atomic` gets really mad if there's an exception even if we catch it and handle it